docs/guides/validation/checkpoints/how-to-run-a-checkpoint-in-python.md

```diff
@@ -1,84 +1,81 @@
 ---
 title: How to run a Checkpoint in python
 ---
+import Prerequisites from '../../../guides/connecting_to_your_data/components/prerequisites.jsx'
 
-import Prerequisites from '../../../guides/connecting_to_your_data/components/prerequisites.jsx';
-
-This guide will help you run a Checkpoint in python.
+This guide will help you run a Checkpoint in Python.
 This is useful if your pipeline environment or orchestration engine does not have shell access.
 
-:::note Prerequisites
-  This how-to guide assumes you have already:
+<Prerequisites>
+
+- Created at least one [Checkpoint](./how-to-create-a-new-checkpoint)
 
-  - Created at least one [Checkpoint](./how-to-create-a-new-checkpoint).
+</Prerequisites>
 
 Steps
 -----
 
-1. First, generate the python with the command:
+1. First, generate the Python with the command:
```

Comment:
If we're capitalizing Python, can we ensure consistency within the entire document?

Reply:
Done!